.github/PULL_REQUEST_TEMPLATE/bug.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Pull Request for a bug
+about: Use this template for bugs
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+<!-- Thank you so much for your time taking to contribute, your work is appreciated! 😃 -->
+
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
+- [ ] Tests for the changes have been added
+- [ ] Docs have been added / updated
+
+## Issue
```

Comment:
Please uses **Issues** with bullet points just after **Description**

Reply:
This would break our contributing guidelines where it specifies that each Pr should fix only one Issue per time.

see [this](https://github.com/geosolutions-it/MapStore2/blame/master/CONTRIBUTING.md#L78)

That's why the issue only